ntoskrnl/fsrtl/fastio.c

```diff
@@ -1795,6 +1795,38 @@ FsRtlReleaseFileForCcFlush(IN PFILE_OBJECT FileObject)
     FsRtlExitFileSystem();
 }
 
+static
```

Comment:
A small documentation may be welcome; in particular, saying that this function returns TRUE when the resource is to be exclusively acquired, and FALSE if not (shared).

Reply:
Added doc & SAL annotations